app/javascript/react/ansibleCatalog/ansibleCatalogItemForm.jsx

```diff
@@ -0,0 +1,375 @@
+    }
+
+    return [
+      {label:__('No') ,value:'no_without_playbook'},
+      {label:__('Yes') ,value:'yes_without_playbook'}
+    ]
+  }
+  const { isLoading, initialValues, isNew } = this.state;
+  return (isLoading? ( <Spinner loading size="lg" />) :
+    (<Form
+      initialValues={initialValues}
+      onSubmit={this.submitForm}
+      mutators={{
+        ...formMutators
+      }}
+      render={({ invalid, values,submitting,pristine, handleSubmit, form: { change,reset, mutators: { push, copyValues } } }) => (
+        <PfForm horizontal onSubmit={handleSubmit}>
+        <div id="flash_msg_div"/>
+          {
+            formFields.map(fields => {
+              if (fields['condition'])
+              {
+                if (values[fields.condition['when']] !== fields.condition.is) {
+                return;
+                }
+              }
+              return renderFormField(fields)
+            })
+          }
+          <TabContainer id="basic-tabs-pf" defaultActiveKey={1}>
+            <div>
+              <Nav bsClass={bsClass}>
+                <NavItem eventKey={1}>
+                  {__('Provisioning')}
+                </NavItem>
+                <NavItem eventKey={2}>
+                  {__('Retirement')}
+                </NavItem>
+              </Nav>
+              <TabContent animation>
+                <TabPane eventKey={1} mountOnEnter={true} unmountOnExit={true}>
+                  <AnsiblePlaybookFields
+                    region={region}
+                    prefix='provisioning'
+                    addExtraVars={push}
+                    formValues={values}
+                    changeField={change}
+                  >
+                  <Field
+                    options={dialogOptions}
+                    label={__('Dialog')}
+                    name='dialogOption'
+                    component={FinalFormSelect}
+                  />
+                  <Condition when="dialogOption" is='existing'>
+                    <Field
+                      name='dialog_id'
+                      label={__('Select Dialog')}
+                      validateOnMount= {true}
+                      validate={required({ msg: __('Required') })}
+                      options={dropdowns.dialogs || []}
+                      component={FinalFormSelect}
+                    />
+                  </Condition>
+                  <Condition when="dialogOption" is='new'>
+                    <Field
+                      name='dialog_name'
+                      label={__('New')}
+                      component={FinalFormField}
+                      validateOnMount={true}
+                      validate={value => {
+                        if (!value) {
+                          return (__('Required'));
+                        }
+                        const exists = dropdowns.dialogs.find((dialog) => {
+                          return dialog.label === value;
+                        });
```

Comment:
You can use shorter syntax here
```javascript
 dropdowns.dialogs.find(dialog => dialog.label === value);
```

Reply:
Good suggestion.  Went ahead and updated it.